updateCli/updateCli.d/jenkins-weekly-jdk11.tpl

```diff
@@ -1,16 +1,13 @@
 source:
   name: "Retrieve latest jenkins weekly version"
-  kind: githubRelease
```

Comment:
it would be possible with a tag format field

weekly:
`[0-9]\.[0-9]+`

lts:
`[0-9]\.[0-9]*\.[0-9]*`

Reply:
Right now we are using the latest release provided by the github API 
https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-the-latest-release

Your suggestion means that I would have to sort every releases from a project